script/openqa-clone-job

```diff
@@ -21,111 +21,114 @@
 
 =head1 openqa-clone-job
 
-openqa-clone-job - clone job from local or remote openQA instances
+Clones a job from the local or another openQA instance. Downloads all assets associated
```

Comment:
what was wrong with "remote"?

Reply:
This was actually "Clone job from another instance." from below and when merging redundant sentences it got mixed up. I can keep using "remote" here.

Reply:
ok, I would prefer that. Because "local or remote" reads better than "local or another".